app/controllers/api/map_controller.rb

```diff
@@ -5,6 +5,13 @@ class MapController < ApiController
     before_action :check_api_readable
     around_action :api_call_handle_error, :api_call_timeout
 
+    before_action :default_format_xml
```

Comment:
We need to be careful about this. Previously in #2065 we ran into problems with accidentally ignoring `Accepts` headers. This approach is different, so perhaps avoids that problem, but we should have test-suite coverage for whatever we are trying to support.

Reply:
I think proper support for `Accept` header is still open. I don't know where the in the processing this information gets lost.

Reply:
default_format_xml now handles an `Accept: application/json` header correctly and returns JSON format in this case.

Reply:
Added a number of unit tests for Accept headers now, so this can probably be set to done now.